docs/1.0. Overview.md

```diff
@@ -27,8 +27,21 @@ Conventions used in the Network Control API that are common with the specificati
 
 The resources that are managed by AMWA IS-06 NMOS Network Control API are specified in the [Data Model](3.0.%20Data%20Model.md).
 
+## LLDP
+
+The Link Layer Discovery Protocol (LLDP), defined by [IEEE 802.1AB](https://standards.ieee.org/standard/802_1AB-2016.html), is a vendor-neutral protocol for network devices (switches) and endpoints to advertise their identity and describe their capabilities.
+
+In order that the broadcast controller and the network controller can communicate about the network topology reliably, this specification strongly recommends LLDP be configured on network devices (switches) and endpoints.
+
+LLDP defines three mandatory parameters including Chassis ID and Port ID.
+
+Where LLDP is implemented, the LLDP identifiers should correspond to the values included in the resources managed by the Network Control API. Otherwise, a broadcast controller may require a manual step to use this API.
+
 ## Relationship with other Specifications
 
 This specification has no dependency on AMWA IS-04 or IS-05. However the overall system is more fully automated with IS-04 and IS-05.
 
+Where LLDP is implemented, this specification recommends that IS-04 Node APIs transmit the Chassis ID and Port ID of each interface, matching the `chassis_id` and `port_id` properties of the `interfaces` in the 'self' resource.
```

Comment:
May not be important, but for cases where LLDP isn't available the intention was to allow Nodes to specify a Chassis ID of `null` and a Port ID matching the MAC they use in gratuitous ARP. It may not may not be worth highlighting this here.

Reply:
That's reasonable in IS-04, but the following bullet-point regarding Nodes' `interfaces.attached_network_device` is currently necessary in order that a BC can discover the Endpoint `attached_network_device` info it needs to populate the **PUT /endpoints/{endpointId}** request? And that requires the Endpoint implement LLDP receipt, so it might as well implement LLDP transmit. 

An alternative would be to make Endpoint `attached_network_device` optional. We'd probably then have to allow a Network Controller to respond with 400 Bad Request if it couldn't identify where the Endpoint is connected from the `port_id` (and `chassis_id` if not `null`) alone? I think this has been discussed previously... @robp-sony @vsachin33

Reply:
Yes, making `attached_network_device` optional would be more consistent with IS-04 where it is also optional. So it would make it easier for a BC to add an Endpoint when the `attached_network_device` information is not available from the IS-04 Registry.

Reply:
This issue now pulled out as #66.